pipeline-modules/infra-service/aws/terraform.yaml

```diff
@@ -146,9 +146,12 @@ template: |
         # replace all '{{ tf_var_name_dash_placeholder }}' placeholders with '-' and quote the value
         # save the result as input variable file for terraform commands
         - printenv | grep "{{ tf_var_name_prefix }}*" | sed 's/^{{ tf_var_name_prefix }}//' | awk -F '=' -v OFS='=' '{ gsub("{{ tf_var_name_dash_placeholder }}", "-", $1); st = index($0,"="); val=substr($0,st+1); print $1, val ~ /^{.*?}$/ || val ~ /^\[.*?\]$/ ? val:"\""val"\"" }' > {{ tf_var_file_path }}
-        # terraform init
         - cd "{{ local_code_path }}"
-        - terraform init > {{ logger('init') }}
+        # terraform state push
```

Comment:
why is this commented out?

Reply:
It is just a comment that is supposed to show up in the generated script to help it break it down into sections. The actual command is below. Same with init.